.azure-pipelines/impacted_area_testing/get-impacted-area.yml

```diff
@@ -1,7 +1,28 @@
 steps:
+- script: |
```

Comment:
If we merge this change, will feature branches skip PR test because of the parameters not defined?

Reply:
Created another PR to test if we do not pass the parameter, and it passed
https://github.com/sonic-net/sonic-mgmt/pull/20905